script/runc.spec

```diff
@@ -0,0 +1,60 @@
+##  Build runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/runc.tar.gz
+##   2. run rpmbuild -ba script/runc-centos.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _bindir /usr/local/bin
+%global _mandir /usr/share/man
+
+Name: runc
+Version: %{_version}
+Release: %{_release}
+Summary: runc is a CLI tool for spawning and running containers according to the OCI specification
+
+License: ASL 2.0
+Source: %{name}-%{version}.tar.gz
+
+URL: https://www.opencontainers.org/
+Vendor: OCI
+Packager: OCI 
+
+BuildRequires: glibc
+BuildRequires: make
+BuildRequires: libseccomp-devel 
+BuildRequires: libselinux-devel
+
```

Comment:
There's no `Requires: criu` here. We should probably include it (even as a recommends or suggests).